docker/Dockerfile

```diff
@@ -0,0 +1,54 @@
+FROM ubuntu:20.04
```

Comment:
instead of ubuntu, have you tried npm based images as base ?

Reply:
npm base image size is 943 MB while Ubuntu 20.04 image is only 72 MB. That is why I choose this.